### PR TITLE
#1563 NUXT - Make color #d32e79 a scss variable

### LINKED ISSUE #1563: Make color #d32e79 a scss variable
Fun little refactoring task 🐰 
Color with hexcode `#d32e79` appears on several occasions throughout the codebase and should be centralised via scss variable in file `derived-variables.scss`

call it `$secondary` or `$primary-light` (any better ideas?)

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main-nuxt** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted screenshot of demonstrated change in this PR

### Optional
- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything works
- [ ] I found edge cases

### What's new?
- [x] PR closes #1563
- [x] Went ahead with primary-light, since its a variation of our primary color

### Had issue bounty label ?
- [x] Fill up your KSM address: 
[Payout](https://kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [ ] My fix has changed **something** on UI, a screenshot for others, is best to understand changes.

### CODE REVIEW

**Review by kkukelka — APPROVED** [review]
LGTM

File: pages/rmrk/collections.vue [hunk 1]
```diff
@@ -307,7 +309,7 @@ export default class Collections extends Vue {
   }
 
   .is-color-pink {
-    color: #d32e79;
+    color: $primary-light;
   }
```
Comment (roiLeo) [inline pages/rmrk/collections.vue:313]: I would move this to global file

File: pages/permafrost/create.vue [hunk 2]
```diff
@@ -533,9 +533,11 @@ export default class PermaMint extends mixins(
...
 
-<style>
+<style lang="scss">
+@import '@/styles/variables';
+
 /* ??? global */
 .message.is-primary .message-body {
-  color: #d32e79 !important;
+  color: $primary-light !important;
 }
```
Comment (roiLeo) [inline pages/permafrost/create.vue:542]: I would move this to global file

File: components/rmrk/Gallery/Gallery.vue [hunk 3]
```diff
@@ -382,7 +384,7 @@ export default class Gallery extends Vue {
   }
 
   .is-color-pink {
-    color: #d32e79;
+    color: $primary-light;
   }
```
Comment (roiLeo) [inline components/rmrk/Gallery/Gallery.vue:388]: I would move this to global file

File: components/rmrk/Gallery/Collections.vue [hunk 4]
```diff
@@ -307,7 +309,7 @@ export default class Collections extends Vue {
   }
 
   .is-color-pink {
-    color: #d32e79;
+    color: $primary-light;
   }
```
Comment (roiLeo) [inline components/rmrk/Gallery/Collections.vue:314]: I would move this to global file

File: components/rmrk/Create/SimpleMint.vue [hunk 5]
```diff
@@ -796,9 +796,11 @@ export default class SimpleMint extends mixins(
...
 
-<style>
+<style lang="scss">
+@import '@/styles/variables';
+
 /* ??? global */
 .message.is-primary .message-body {
-  color: #d32e79 !important;
+  color: $primary-light !important;
 }
```
Comment (roiLeo) [inline components/rmrk/Create/SimpleMint.vue:805]: I would move this to global file

File: components/rmrk/Create/Admin/AdminPanel.vue [hunk 6]
```diff
@@ -295,8 +295,10 @@ export default class AdminPanel extends mixins(
...
 </script>
 
-<style>
+<style lang="scss">
+@import '@/styles/variables';
+
 .message.is-primary .message-body {
-  color: #d32e79 !important;
+  color: $primary-light !important;
 }
```
Comment (roiLeo) [inline components/rmrk/Create/Admin/AdminPanel.vue:303]: I would move this to global file

**Review by roiLeo — COMMENTED** [review]
Some of the rules are duplicated could you try to move them outside component?

**Review by roiLeo — APPROVED** [review]
lgtm

**Comment by yangwao** [comment]
Perfect, I’ve sent the payout https://kusama.subscan.io/extrinsic/0x90a446bc3f095476407a3954d7e47adb80121a2e4fb69cca7410791500646e0f

Let’s grab another issue and get rewarded! 
https://github.com/kodadot/nft-gallery/issues/

**Pull request merged**